config/develop/example-dev-project.j2

```diff
@@ -1,28 +1,40 @@
-stack_name: example-dev-project
+{% set stack_name = "example-dev-project" %}
+{% set read_write_users = ["bruno.grande@sagebase.org"] %}
+{% set read_only_users = ["tess.thyer@sagebase.org", "thomas.yu@sagebase.org"] %}
 
-stack_tags:
-  Department: IBC
-  Project: Infrastructure
-  OwnerEmail: nextflow-admins@sagebase.org
+# DO NOT EDIT ANYTHING BELOW THIS LINE -----------------------------------------------------------
+
+template_path: tower-project.yaml
+stack_name: {{ stack_name }}
+dependencies:
+  - common/nextflow-forge-iam-policy.yaml
+  - common/nextflow-launch-iam-policy.yaml
 
 parameters:
+  {% if read_write_users %}
   S3ReadWriteAccessArns:
-    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/bruno.grande@sagebase.org
+  {% for user in read_write_users %}
+    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/{{ user }}
+  {% endfor %}
+  {% endif %}
+  {% if read_only_users %}
   S3ReadOnlyAccessArns:
-    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/tess.thyer@sagebase.org
-
-  # DO NOT EDIT ANYTHING BELOW THIS LINE ------------------------------------------------------------------------
-
+  {% for user in read_only_users %}
+    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/{{ user }}
+  {% endfor %}
+  {% endif %}
   AccountAdminArns:
     - {{stack_group_config.sso_admin_role.arn}}
     - !stack_output_external workflows-nextflow-ci-service-account::ServiceRoleArn
   TemplateRootUrl: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com'
   TowerForgePolicyArn: !stack_output_external nextflow-forge-iam-policy::NextFlowForgePolicyArn
   TowerLaunchPolicyArn: !stack_output_external nextflow-launch-iam-policy::NextFlowLaunchPolicyArn
 
-template_path: tower-project.yaml
+stack_tags:
+  Department: IBC
+  Project: Infrastructure
+  OwnerEmail: nextflow-admins@sagebase.org
 
-dependencies:
-  - develop/nextflow-vpc.yaml
-  - common/nextflow-forge-iam-policy.yaml
-  - common/nextflow-launch-iam-policy.yaml
+hooks:
+  after_launch:
+    - !cmd bin/configure-tower-project.py -s {{ stack_name }} -m {{ read_write_users | join(" ") }} -v {{ read_only_users | join(" ") }}
```

Comment:
This is a good idea. In this case, I think we'll be [moving to a CI step](https://github.com/Sage-Bionetworks-Workflows/aws-workflows-nextflow-infra/pull/62#pullrequestreview-750510456) instead of a post-launch hook. That allows us to only run the common steps once (like creating the organizations) and skip configs that haven't changed.